Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -106,6 +117,31 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+/// Returns an identity matrix or a batch of matrices.
+///
+/// - Parameters:
+///   - numRows: A non-negative integer giving the number of rows in each batch matrix.
+///   - numCols: A non-negative integer giving the number of rows in each batch matrix.
+///   - batchShape: A list of integers. The returned tensor will have leading batch dimensions
+///     of this shape.
+public func eye<Scalar: Numeric>(
+    numRows: Int,
```

Comment:
Variables related to the "number of things" are named `xxxCount` in Swift, e.g. [`Array.count`](https://developer.apple.com/documentation/swift/array/2943906-count) and [`Tensor.scalarCount`](https://www.tensorflow.org/swift/api_docs/Structs/Tensor#scalarcount).
```suggestion
    rowCount: Int,
```